url.bs

```diff
@@ -2858,10 +2866,19 @@ method, when invoked, must run these steps:
  <li><p>Run the <a for=URLSearchParams>update steps</a>.
 </ol>
 
-<p>The
-<dfn method for=URLSearchParams><code>has(<var>name</var>)</code></dfn>
-method, when invoked, must return true if there is a name-value pair whose name is <var>name</var>
-in <a for=URLSearchParams>list</a>, and false otherwise.
+<hr>
+
+<p>The <dfn method for=URLSearchParams><code>sort()</code></dfn> method, when invoked, must run
+these steps:
+
+<ol>
+ <li><p>Lexicographically sort all name-value pairs, if any, by name, while preserving the relative
```

Comment:
Having looked at https://tc39.github.io/ecma262/#sec-abstract-relational-comparison I think we want code unit order, not code point order. So something like:

> Sort all name-value pairs, if any, by name on code unit, while preserving the relative order between duplicate names, if any.